lib/docassemble/setup.js

```diff
@@ -20,7 +20,17 @@ const setup = async () => {
 
   // Create a project on the da server's account with a unique name then
   // store it locally for later deleting the project. 
-  let project_name = await da_i.loop_to_create_project();
+  let project_name = null;
+  // Explicit try-catch prevents "Unhandled promise rejection error"
+  // avoids showing confusing node errors to users that they shouldn't have to fix
+  try {
+    project_name = await da_i.loop_to_create_project();
+  } catch (error) {
+    log.error({type: `setup`, pre: `something happened when creating docassemble project`, data: JSON.stringify(error, null, 4)})
```

Comment:
```suggestion
    log.error({type: `setup`, pre: `an unexpected error occurred while trying to create a docassemble project in the user's account`, data: JSON.stringify(error, null, 4)})
```

I'm just trying to add something about an error or something about specifically not being able to create the project to be more specific than something happening. I imagine folks like Caroline being a bit more annoyed about the lack of specificity. I do wish we could add a message about what their next step might be. Even something like looking at the previous lines in the log, checking the user account, looking at the docs, or contacting us. Maybe that has to wait till we fill out the docs with more info about specific error messages.